src/vip-framework/checks/checkCorePoolComptroller.ts

```diff
@@ -0,0 +1,117 @@
+import { expect } from "chai";
+import { Contract, Signer } from "ethers";
+import { ethers } from "hardhat";
+import mainnet from "@venusprotocol/venus-protocol/networks/mainnet.json"
+import testnet from "@venusprotocol/venus-protocol/networks/testnet.json"
+
+import COMPTROLLER_ABI from "../abi/comptroller.json";
+import { setMaxStalePeriodInChainlinkOracle } from "../../utils";
+import { impersonateAccount, mine } from "@nomicfoundation/hardhat-network-helpers";
+import ERC20_ABI from "../abi/erc20.json";
+import VTOKEN_ABI from "../abi/vToken.json";
+import { parseUnits } from "ethers/lib/utils";
+
+let vETH_ADDRESS = mainnet.Contracts.vETH
+let vUSDT_ADDRESS = mainnet.Contracts.vUSDT
+let USDT = mainnet.Contracts.USDT;
+let ETH = mainnet.Contracts.ETH;
+let NORMAL_TIMELOCK = mainnet.Contracts.Timelock;
+let XVS = mainnet.Contracts.XVS;
+let COMPTROLLER = mainnet.Contracts.Unitroller;
+let LENS = mainnet.Contracts.ComptrollerLens;
+let ETH_FEED = "0x9ef1B8c0E4F7dc8bF5719Ea496883DC6401d5b2e";
+let USDT_FEED = "0xB97Ad0E74fa7d920791E90258A6E2085088b4320";
+let ACCOUNT = "0x5a52E96BAcdaBb82fd05763E25335261B270Efcb";
+let CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+
+if (process.env.FORK_TESTNET === "true") {
```

Comment:
I think this would be now `FORKED_NETWORK === "bsctestnet"`

Reply:
I would integrate these addresses in the new file `vips/src/networkAddresses.ts`. And then we could use it this way:

```
import { NETWORK_ADDRESSES } from "../networkAddresses";
const { NORMAL_TIMELOCK, CHAINLINK_ORACLE, ... }  = NETWORK_ADDRESSES[process.env.FORKED_NETWORK]
```

Reply:
Configuration, like the pool ID, could be added to `src/networkConfig.ts`